src/CVCard.ts

```diff
@@ -0,0 +1,81 @@
+import { html, nothing, TemplateResult } from "lit-html";
+import { styleMap } from "lit-html/directives/style-map";
+import {
+  fullWidth,
+  heading,
+  paddingSmall,
+  textCenter, textLeft,
+  textGray,
+} from "./baseStyles";
+import { ProfilePresentation } from "./presenter";
+import { CVPresentation } from "./CVPresenter";
+
+const styles = {
+  image: styleMap(fullWidth()),
+  intro: styleMap({ ...textGray(), ...textCenter() }),
+  card: styleMap({}),
+  info: styleMap({ ...paddingSmall(), ...textLeft()}),
+};
+
+export const CVCard = (profileBasics: ProfilePresentation, cvData: CVPresentation): TemplateResult => {
+  const { rolesByType, skills } = cvData
+  const nameStyle = styleMap({
+    ...heading(),
+    // "text-decoration": "underline",
+    "color": profileBasics.highlightColor, // was "text-decoration-color"
+  });
+  function renderRole (role) {
```

Comment:
I feel like we should have the function outside of the main function and just call them.  It gets us into a bit of trouble if we want to do unit testing.